repotracker/wrappers.go

```diff
@@ -0,0 +1,100 @@
+package repotracker
```

Comment:
Is it feasible to have tests for these functions?

Reply:
we're really just refactoring existing code with very few modifications, and there's no real way to isolate these functions from their underpinnings (which has always been the case.)

I wanted to avoid rewriting the repotracker internals as part of this project, so I'm leaning towards no on this.